src/STRPAK.CPP

```diff
@@ -1105,7 +1121,8 @@ char* strPluralize(				// form plural of a word
 	else
 	{
 		bool bException = false;
-		for (int i=0; excpTbl[ i].wordSingular; i++)
+		int i;
+		for (i=0; excpTbl[ i].wordSingular; i++)
```

Comment:
Wow, it's weird that the subsequent use of "i" was ever valid!

Reply:
Yes, I have found that you can get away with a lot of little things like this in MSVC.

Reply:
I've seen this used in other places too. VS even highlights it as problematic, but never complains!